selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -99,6 +101,10 @@ void Networking::refresh() {
       return;
     }
   }
+  wifi->requestScan();
+}
+
+void Networking::refresh() {
   wifiWidget->refresh();
   an->refresh();
 }
```

Comment:
We can leave out `firstRefresh` if we believe that NM will never send us a scan complete signal (`LastScan`) without us requesting it, since we don't check that Networking is visible here (so we can update once in the background). It can block UI for 70 ms+ if NM ever does though